zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -1272,6 +1272,44 @@ paths:
               schema:
                 $ref: "#/components/schemas/ProblemDetail"
 
+  /elements/{elementKey}/variabels:
+    post:
+      tags:
+        - Variables
+      summary: Updates element variables
+      description: |
+        Updates all the variables of a particular scope (e.g. process instance, flow element
+        instance) with the given variable data.
```

```suggestion
        Updates all the variables of a particular scope (e.g. process instance, flow element instance) with the given variable data.
```

:x: This is the issue I've mentioned about our docs generator choosing exactly the first line for the page's meta description, which results in an incomplete sentence in cases like this. 

💭 I don't like how frequently this idiosyncrasy of our docs generator is already making an appearance in my reviews, especially since I suspect you all are just using your editor to auto-wrap these descriptions. See my & @tmetzke's related conversation at https://github.com/camunda/camunda/pull/21787#discussion_r1745757112.

---

I need to pay more attention, thanks 🙇